src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -3489,7 +3599,7 @@ internal void SendAttention(bool mustTakeWriteLock = false)
                             }
                         }
 #if DEBUG
-                    }
+                }
 #endif
```

Comment:
Looks like this came back.

Reply:
The merge got screwed up and I fixed it up. Since it's just formatting i suggest we leave as it and deal with it later.